bigtable/client/client_options.h

```diff
@@ -48,11 +48,127 @@ class ClientOptions {
   const grpc::ChannelArguments channel_arguments() const {
     return channel_arguments_;
   }
-  ClientOptions& SetChannelArguments(grpc::ChannelArguments channel_arguments) {
+  ClientOptions &SetChannelArguments(grpc::ChannelArguments channel_arguments) {
     channel_arguments_ = channel_arguments;
     return *this;
   }
 
+  /*
+   * Set Compression Algorithm for channel.
+   *
+   * Please look at
+   * https://grpc.io/grpc/cpp/classgrpc_1_1_channel_arguments.html#aaf332071bbdff5e4a7f3352f4ad564d5
+   * for more details.
```

Comment:
Is there a way to do that without including the SHA-1 in the link?

Reply:
We can omit SHA-1 but after that user will have to navigate to setter on his own.

Reply:
Is the SHA-1 stable?  What happens when grpc.io regenerates their docs?

Reply:
How about:

> Please see the docs for `grpc::ChannelArguments::SetCompressionAlgorithm()` on https://grpc.io/grpc/cpp/classgrpc_1_1_channel_arguments.html for more details.

Similarly for other methods here — tell the user what to look for rather than an opaque hash.

Thoughts?

Reply:
Sounds good to me. Most probably they must be generating this SHA-1 using the contents.